dev.zed.Zed.yaml

```diff
@@ -52,6 +52,17 @@ modules:
         sha256: 29df50974a90987af694c0fb8926a6b366e68cacd8abd813817cfe1eb5d54524
         url: https://download.gnome.org/sources/gcr/3.34/gcr-3.34.0.tar.xz
 
+  - name: openssl
+    buildsystem: simple
+    build-commands:
+      - ./config --prefix=/app --openssldir=/app/ssl -Dshared=true
+      - make
+      - make install
+    sources:
+      - type: archive
+        url: https://www.openssl.org/source/openssl-3.2.0.tar.gz
+        sha256: "14c826f07c7e433706fb5c69fa9e25dab95684844b4c962a2cf1bf183eb4690e"
+
```

Comment:
Is there a way in flatpak use an existing openssl flatpak?

Reply:
I understand that there is no extension that provides openssl support.

Reply:
And then does a flatpak internal dependencies are checked for update by flathubbot? 

openssl is known to expose from time to time serious flaws needing update.